client/src/Components/ResultBox.js

```diff
@@ -1,8 +1,63 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
 
 
-const ResultBox= () => {
-  return <input className="resultBox" type="text" id="result" name="result"></input>;
-};
+function ResultBox({ products, isSelected_DropDown }) {
+  const [productData, setProductData] = useState([]);
 
+  useEffect(() => {
+    setProductData(products);
+  });
+
+  const product_handling = productData
+    .filter(data => data.productcategory === isSelected_DropDown)
+    .map(data => data.handling);
+  console.log('Output handling', product_handling);
```

Comment:
for the PRs try to clear out the console.logs.

Reply:
Done.